### PR TITLE
Change `if` keyword to `assert`

### PR DESCRIPTION
**Pull request by dandclark**
Change `if` keyword to `assert` in README and spec.
Add `[no LineTerminator here]` prior to `AssertClause`.

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -23,12 +23,12 @@ <h1>Syntax</h1>
     ImportDeclaration :
       `import` ImportClause FromClause `;`
       `import` ModuleSpecifier `;`
-      <ins>`import` ImportClause FromClause IfClause `;`</ins>
-      <ins>`import` ModuleSpecifier IfClause `;`</ins>
-      <ins>`export` ExportFromClause FromClause IfClause `;`</ins>
+      <ins>`import` ImportClause FromClause [no LineTerminator here] AssertClause `;`</ins>
+      <ins>`import` ModuleSpecifier [no LineTerminator here] AssertClause `;`</ins>
+      <ins>`export` ExportFromClause FromClause [no LineTerminator here] AssertClause `;`</ins>
```
Comment (littledan) [inline spec.html:28]: Unrelated editorial error: This whole line should actually be part of ExportDeclaration, not ImportDeclaration.

**Comment by dandclark** [comment]
> This change LGTM _if_ we want to rename `if` to `assert`. I think we can decide whether we want to do this change during the TC39 meeting, and land if so.
> 
> We may want to rename [[Conditions]] to [[Assertions]] and generally rename this proposal (!) to "import assertions", but this can be a later PR.

Sounds good, I'll just leave this un-merged for now.
Separately I created #81 to fix the editorial issue you pointed out.

File: spec.html (hunk 1 above)
Comment (dandclark) [reply spec.html:31]: Resolved in #81, and rebased.

**Comment by xtuc** [comment]
As discussed during the last TC39 meeting, it seems that people are in favor of switching to assert

**Comment by ljharb** [comment]
Should the proposal be renamed as well, to "import assertions"?

**Comment by dandclark** [comment]
> Should the proposal be renamed as well, to "import assertions"?

I think so, and @littledan also pointed out that the [[Conditions]] field should probably also be renamed.  I filed #87 and I will make these changes if there are no objections.

**Pull request merged**